src/parser.sid

```diff
@@ -388,5 +476,5 @@
 
 %entry%
 
-	file;
+	file, schema;
```

Comment:
Fine.

Reply:
I don't particularly like this, but I couldn't figure out how to build a subschema without making a ton of changes that ultimately just pass the current schema around.  This seems to do the same thing with less boilerplate, although it's a lot less explicit.